docs/cli/kops_update.md

```diff
@@ -24,6 +24,6 @@ Update clusters
 
 ### SEE ALSO
 * [kops](kops.md)	 - kops is kubernetes ops
-* [kops update cluster](kops_update_cluster.md)	 - Update cluster
+* [kops update cluster](kops_update_cluster.md)	 - Execute the creation or modification of a cloud or cluster resources.
```

Comment:
Drop "a"?  Also, drop "Execute the" IMO

"Create or modify cloud or cluster resources."